_toc.yml

```diff
@@ -15,33 +15,43 @@ parts:
       sections:
       - file: environment_set_up/Help_AWS_Credentials
       - file: environment_set_up/Start_Dask_Cluster_Nebari  
-      #- file: environment_set_up/Start_Dask_Cluster_PangeoCHS
+      - file: environment_set_up/Start_Dask_Cluster_PangeoCHS
```

Comment:
This is not going to work.  See https://github.com/hytest-org/hytest/issues/274#issuecomment-1502216128

Reply:
The cluster notebooks for pangeo, denali, etc only really work on those environments. But the book is built on nebari. So the attempt to execute `Start_Dask_Cluster_PangeoCHS` will fail, which means the book won't build. 

There are workarounds -- see the comment linked above -- but the executable notebooks as-written will not build successfully on nebari. 

And we can't move the whole notebook to denali to run -- all of the workflows are designed and tested around nebari execute environment.  It is far easier to do the workaround for these special case notebooks rather than for the rest of the repo.  IMO.

Reply:
Ah thanks - sorry I missed that!

Reply:
addressed in #290 - now since the md and ipynb files have the same name, which will JB pull? Is this a case where we want to add the .md extension?

Reply:
After a bit more research, I think I've found a better solution: 

We can include the `.ipynb`, but direct the compiler not to run it. This is a capability that I only recently became aware of.  This would simplify the problem and also keep the repo cleaner. 

To discuss.